solutions/usaco-919.mdx

```diff
@@ -54,47 +59,40 @@ int main() {
 import java.util.*;
 import java.io.*;
 
-public class paintbarn {
-	static Kattio io;
-	static {
-		try {
-			io = new Kattio("paintbarn");
-		} catch(IOException e) {}
-	}
-
-	static int N, K, A=0;
-	static int[][] psum;
+public class PaintBarn {
+	private static final int WIDTH = 1000;
```

Comment:
any reason this is private?

Reply:
i keep access levels as limited as possible

Reply:
I would recommend making the variable public. I think a lot of students probably don't know what the private keyword means in Java, and clarity of code takes precedence over good software engineering practices, in my opinion.

Reply:
if they don't know what private means why would they know what public means

Reply:
well you don't need public or private you just do static int ...

and also you type public static void every single time, you almost never type private in competitive programming

Reply:
who's "you" now

Reply:
it is not necessary to do public or private for global variables but I don't think this is a required change

Reply:
yeah imo just remove `private`, it doesn't really serve a purpose and may confuse people newer to java

Reply:
don't think a simple access modifier would confuse people?

Reply:
not too confusing, but better to do what's more common (not write `private`)